lapis2/src/main/kotlin/org/genspectrum/lapis/openApi/OpenApiDocs.kt

```diff
@@ -249,8 +249,19 @@ private fun lapisResponseSchema(dataSchema: Schema<Any>) =
     Schema<Any>().type("object").properties(
         mapOf(
             "data" to Schema<Any>().type("array").items(dataSchema),
+            "info" to infoResponseSchema(),
         ),
-    ).required(listOf("data"))
+    ).required(listOf("data", "info"))
+
+private fun infoResponseSchema() =
+    Schema<Any>().type("object")
+        .description("Information about LAPIS.")
+        .properties(
+            mapOf(
+                "dataVersion" to Schema<String>().type("string").description("The data version of the data in SILO.")
```

Comment:
`LapisResponseAnnotation` already has a description for that. I think we should unify that. And maybe point out, that it's actually the same value?